third_party/miniz-3.0.2/miniz.h

```diff
@@ -1002,7 +1072,11 @@ typedef enum {
     MZ_ZIP_FLAG_WRITE_ZIP64 = 0x4000,               /* always use the zip64 file format, instead of the original zip file format with automatic switch to zip64. Use as flags parameter with mz_zip_writer_init*_v2 */
     MZ_ZIP_FLAG_WRITE_ALLOW_READING = 0x8000,
     MZ_ZIP_FLAG_ASCII_FILENAME = 0x10000,
-    MZ_ZIP_FLAG_DO_NOT_COMPUTE_CRC32 = 0x20000, /* don't compute the crc32 of file data that's being added. */
+    /*After adding a compressed file, seek back
+    to local file header and set the correct sizes*/
+    MZ_ZIP_FLAG_WRITE_HEADER_SET_SIZE = 0x20000,
+    MZ_ZIP_FLAG_DO_NOT_COMPUTE_CRC32 = 0x30000, 
```

Comment:
I suspect this might have to be 
```suggestion
    MZ_ZIP_FLAG_DO_NOT_COMPUTE_CRC32 = 0x80000, 
```
for the flag to work properly